turbinia/task_utils.py

```diff
@@ -173,6 +173,7 @@ def task_deserialize(input_dict):
   # empty Task is instantiated and we don't want to overwrite it.
   input_dict.pop('output_manager')
   task.__dict__.update(input_dict)
+  task.start_time = datetime.strptime(input_dict['start_time'], DATETIME_FORMAT)
```

Comment:
This may answer some of the above questions regarding when Task gets `start_time` but leaving above to confirm.

Reply:
I think the most relevant line is in `TurbiniaTask.__init__()` where this gets initialized

Reply:
Ack